packages/core/sdk/src/auth/providers/internal/StitchAuthResponseCredential.ts

```diff
@@ -0,0 +1,31 @@
+import AuthInfo from "../../internal/AuthInfo";
+import ProviderCapabilities from "../../ProviderCapabilities";
+import StitchCredential from "../../StitchCredential";
+
+export default class StitchAuthResponseCredential implements StitchCredential {
+    /**
+     * The name of the authentication provider that this credential will be used to authenticate with.
+     */
+    public readonly providerName: string;
+
+    /**
+     * The type of the authentication provider that this credential will be used to authenticate with.
+     */
+    public readonly providerType: string;
+
+    /**
+     * A `ProviderCapabilities` object describing the behavior of this credential when logging in.
+     */
+    public providerCapabilities: ProviderCapabilities;
+
+    /**
+     * The contents of this credential as they will be passed to the Stitch server.
+     */
+    public readonly material: { [key: string]: string };
+
+    /**
```

Comment:
The contents aren't really passed to the stitch server. AuthInfo is a result to clients, not to the server. we only send access and refresh tokens back.

Reply:
👍

Reply:
This wasn't updated